spec/integration/ClientSpec.php

```diff
@@ -695,4 +698,33 @@ function it_receives_the_expected_response_when_looking_up_received_texts() {
           $received_text['content']->shouldBeString();
       }
     }
+
+    function it_receives_the_expected_response_when_getting_a_pdf_for_a_letter_notification() {
+
+      // Requires the 'it_receives_the_expected_response_when_sending_a_letter_notification' test to have completed successfully
```

Comment:
Not a massive fan of integration tests relying on success of a previous test but you've added a comment here which is probably enough to satisfy me

Reply:
me neither, but it a) reduces the time it takes to run the tests, and b) replicates existing pattern from `it_receives_the_expected_response_when_looking_up_an_email_notification` and `it_receives_the_expected_response_when_looking_up_an_sms_notification`